prodtests/full-system-test/aggregator-workflow.sh

```diff
@@ -148,7 +148,10 @@ fi
 if [[ $AGGREGATOR_TASKS == BARREL_TF ]] || [[ $AGGREGATOR_TASKS == ALL ]]; then
   # PrimVertex
   if [[ $CALIB_PRIMVTX_MEANVTX == 1 ]]; then
-    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=55000"
+    if [[ -z $TFPERSLOTS_MEANVTX ]]; then TFPERSLOTS_MEANVTX=55000; fi
+    DELAYINTFS_MEANVTX="10"
+    MAXDELAY_INSLOTFRACTION_MEANVTX=`awk "BEGIN {print $DELAYINTFS_MEANVTX/$TFPERSLOTS_MEANVTX}"`
+    add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX --max-delay $MAXDELAY_INSLOTFRACTION_MEANVTX"
```

Comment:
After checking mean-vertex code I see that there is no --max-delay but only `MeanVertexParams::maxTFdelay`.
Should not it be 
```
add_W o2-calibration-mean-vertex-calibration-workflow "" "MeanVertexCalib.tfPerSlot=$TFPERSLOTS_MEANVTX;MeanVertexCalib.maxTFdelay=$MAXDELAY_INSLOTFRACTION_MEANVTX"
```
?